api/api.py

```diff
@@ -597,3 +597,19 @@ def enable_feature(feature_name: str) -> None:
   Args:
     feature_name: name of the feature to enable
   """
+  pass
+
+def local_resource(name: str, cmd: str, deps: Union[str, List[str]] = None, trigger_mode: TriggerMode = TRIGGER_MODE_AUTO) -> None:
+  """Configures `cmd` to run on the _host_ machine, not in a remote cluster.
+
+  If `deps` is set then `cmd` is only run if one of the files specified changes.
+
+  If `trigger_mode` is set then the local resource obeys the trigger mode.
```

Comment:
this seems kinda tautological, imo can cut this and just explain in the `args` section below